# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bcron/bcron-0.10-make-install.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2013-04-18 13:41:10.011910953 +0200
+++ ./Makefile	2013-04-18 13:58:39.876580718 +0200
@@ -4,6 +4,14 @@
 
 SHELL=/bin/sh
 
+DESTDIR=
+PREFIX=/usr/local
+BINDIR=$(PREFIX)/bin
+MANDIR=$(PREFIX)/share/man
+MAN1DIR=$(MANDIR)/man1
+MAN5DIR=$(MANDIR)/man5
+MAN8DIR=$(MANDIR)/man8
+
 DEFAULT: all
 
 all: libraries programs docs
@@ -109,8 +117,21 @@
 docs: bcron.info bcron.html crontab.5.html bcrontab.1.html bcron-spool.8.html bcron-sched.8.html bcron-exec.8.html bcron-start.8.html bcron-update.8.html
 
 install: INSTHIER conf-bin conf-man
-	bg-installer -v <INSTHIER
-	bg-installer -c <INSTHIER
+	install -m 0755 bcron-exec $(DESTDIR)$(BINDIR)/
+	install -m 0755 bcron-sched $(DESTDIR)$(BINDIR)/
+	install -m 0755 bcron-spool $(DESTDIR)$(BINDIR)/
+	install -m 0755 bcron-start $(DESTDIR)$(BINDIR)/
+	install -m 0755 bcron-update $(DESTDIR)$(BINDIR)/
+	install -m 0755 bcrontab $(DESTDIR)$(BINDIR)/
+	ln -snf bcrontab $(DESTDIR)$(BINDIR)/crontab
+	install -m 0644 bcrontab.1 $(DESTDIR)$(MAN1DIR)/
+	ln -snf bcrontab.1 $(DESTDIR)$(MAN1DIR)/crontab.1
+	install -m 0644 crontab.5 $(DESTDIR)$(MAN5DIR)/
+	install -m 0644 bcron-exec.8 $(DESTDIR)$(MAN8DIR)/
+	install -m 0644 bcron-sched.8 $(DESTDIR)$(MAN8DIR)/
+	install -m 0644 bcron-spool.8 $(DESTDIR)$(MAN8DIR)/
+	install -m 0644 bcron-start.8 $(DESTDIR)$(MAN8DIR)/
+	install -m 0644 bcron-update.8 $(DESTDIR)$(MAN8DIR)/
 
 job.o: compile job.c bcron.h
 	./compile job.c
